docs/api/proposed_endpoints.md

```diff
@@ -507,3 +527,420 @@ Status: 200 OK
 }
 ```
 
+## Send notification to individual speaker
+
+```
+POST /api/proposals/:id/notify (requires authentication)
+```
+
+**Response**
+Status: 200 OK
+```json
+{
+    "message": "Notification of proposal accepted will be sent to the speaker."
+}
+```
+
+__Note__:
+- Notification will be sent only if the `status` of the proposal is `accepted`.
+
+## Send notification to all the speakers
+
+```
+POST /api/proposals/notify (requires authentication)
+```
+
+**Response**
+Status: 200 OK
+```json
+{
+    "message": "Notification of proposal accepted will be sent to all the speakers."
+}
+```
+
+__Note__:
+- Notification will be sent only if the `status` of the proposal is `accepted`.
+
+# Swags
+
+**Parameters**
+
+Name              | Data Type | Required | Default Value  | Discription
+------------------|-----------|----------|----------------|--------------------
+id                | UUID      | false    | _uuid_         | Unique ID for the swag.
+item              | UUID      | true     | ''             | ID for the swag item.
+description       | text      | true     | ''             | Description of the swag.
+image             | text      | false    | ''             | URL of the image of the swag.
+created_at        | datetime  | false    | _datetime_     | Time of creation.
+modified_at       | datetime  | false    | _datetime_     | Time of last modification.
+
+__Note__:
+- _uuid_: Randomly generated Universally Unique IDentifier (UUID).
+- _datetime_: Date and time when `Swag` is created.
+
+## Create a swag
+
+```
+POST /api/swags (requires authentication)
+```
+
+**Request**
+```json
+{
+    "item": "12abcff0-bcbc-01fc-abcd-01012bc4e0b1",
+    "description": "sponsered by someone"
+}
+```
+
+**Response**
+Status: 201 Created
+```json
+{
+    "id": "0f342ac1-ac32-4bd1-3612-efa32bc3d9a0",
+    "item": "12abcff0-bcbc-01fc-abcd-01012bc4e0b1",
+    "description": "sponsered by someone",
+    "image": null,
```

Comment:
Make it as URL so it is pretty evident for someone integrating the API on what to expect.